src/Rules/Variables/ThrowTypeRule.php

```diff
@@ -0,0 +1,69 @@
+<?php declare(strict_types = 1);
+
+namespace PHPStan\Rules\Variables;
+
+use PhpParser\Node;
+use PHPStan\Analyser\Scope;
+use PHPStan\Type\MixedType;
+use PHPStan\Type\ObjectType;
+use PHPStan\Type\TypeCombinator;
+use PHPStan\Type\VerbosityLevel;
+
+class ThrowTypeRule implements \PHPStan\Rules\Rule
+{
+
+	/** @var bool */
+	private $reportMaybes;
+
+	/** @var bool */
+	private $checkNullables;
```

Comment:
Please use RuleLevelHelper and separate the rule in two. Thanks.

Reply:
(Checking possibly nullables on L7.)

Reply:
Rebased to master and reused `RuleLevelHelper::findTypeToCheck()`. Check it please